Comparison shopping engine/Forms/MainForm.cs

```diff
@@ -134,12 +135,31 @@ private void About(object sender, EventArgs e)
 
         private void Search(object sender, EventArgs e)
         {
+            if(Initializer.DoneWithDatabase)
+                FetchSearched();
```

Comment:
Kokia šito void prasmė?

Reply:
Kaip suprasti void?
Cia tokiu atvėju jeigu is duombazės viską parsitraukė, jis vėl kreiptus į duombazę ir gautų naujus itemus, kurie galejo būti pridėti kito vartotojo. Kitaip sakant visada turetumem naujausia duombazes versija kai nuspaudziame search.

Reply:
Supratau. Sakydamas void turėjau omeny funkcija.